### ISSUE TITLE
Documentation mismatch with Zoom documentation

### ISSUE THREAD

**Issue by QiUnknown** [MacsiDigital/laravel-zoom#15]
From the readme, regarding the creation of meetings:

```
$meeting = $user->meetings()->create([
    	'type' => '2',
    	'start_time' => '2019-06-29T20:00:00Z'
]);
```

This code, when I was running in a command, will get a 405 from zoom server. I read through the Zoom docs, and without the trailing 'Z', it would go through without issue. Please update the doc at your leisure. Thanks for the great library.

**Comment by janpaulalmanoche** [comment]
the same with me.im having a 400 status code

$user = $zoom->user->create([
        'name' => 'Test Name',
        'first_name' => 'First Name',
        'last_name' => 'Last Name',
        'email' => 'test@test.com',
        'password' => 'secret',
        'type' => 1
    ]);
    $meeting = $user->meetings()->create([
    	'type' => '2',
    	'start_time' => '2019-06-29T20:00:00Z'
    ]);
    $registrant = $meeting->registrants()->create([
    	'email' => 'registratn@domain.com',
    	'first_name' => 'Test',
    	'last_name' => 'Registrant'
    ]);

**Comment by colinhall17** [comment]
Hi

Apologies in the delay in coming back to you, when I created this package it was a quick build for a project I needed it for and I made it open source in case it would help someone. 

With the increased use of Zoom around the world its gaining some traction, so I've built 3 new versions for Laravel 5.5 upwards, make sure you select the correct version for your version of Laravel.

The new version uses Laravel date casting so on creation pass in a Carbon Instance and the API should handle the rest.

Open a new issue with any problems.
